deploy/csi-operator-ubuntu-18.04.yaml

```diff
@@ -322,9 +328,10 @@ spec:
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-cluster-driver-registrar
           image: quay.io/k8scsi/csi-cluster-driver-registrar:v1.0.1
+          imagePullPolicy: IfNotPresent
           args:
             - "--v=5"
-            - "--driver-requires-attachment=false"
+            - "--driver-requires-attachment=true"
```

Comment:
Hi, the flag `--driver-requires-attachment` seems will be  deprecated in the next version. the container `csi-cluster-driver-registrar` runs well by using the v1.0.1. 
But when I use the image built from master. the container will crash. And the log shows
```
root@ubuntu:/home/wzh/goProject/src/github.com/openebs/cstor-csi-deploy# kubectl logs openebs-cstor-csi-controller-0 -n kube-system -c csi-cluster-driver-registrar
flag provided but not defined: -driver-requires-attachment                                                                               
Usage of /csi-cluster-driver-registrar:                                                                                                                                                          -alsologtostderr                                                                                                                                 
        log to standard error as well as files                                                                                                          
  -connection-timeout duration                                                                                                         
        The --connection-timeout flag is deprecated                                                                                                                
  -csi-address string                                                                                                        
        Address of the CSI driver socket. (default "/run/csi/socket")                                                                      
  -kubeconfig string                                                                                                                                
        Absolute path to the kubeconfig file. Required only when running out of cluster.                                                                                                       
  -log_backtrace_at value                                                                                                                               
        when logging hits line file:N, emit a stack trace                                                                                           
  -log_dir string                                                                                                                                  
        If non-empty, write log files in this directory                                                                                                                                        
  -log_file string                                                                                                      
        If non-empty, use this log file                                                                                                                 
  -logtostderr                                                                                                                                           
        log to standard error instead of files (default true)                                                                                         
  -pod-info-mount                                                                                                                                    
        This indicates that the associated CSI volume driverrequires additional pod information (like podName, podUID, etc.) during mount.When set to true, Kubelet will send the followings po
d information during NodePublishVolume() calls to the driver as VolumeAttributes:- csi.storage.k8s.io/pod.name: pod.Name     
        - csi.storage.k8s.io/pod.namespace: pod.Namespace                                                                                                
        - csi.storage.k8s.io/pod.uid: string(pod.UID)                                                       
  -skip_headers                                                                                                                                 
        If true, avoid header prefixes in the log messages                                                                                                
  -stderrthreshold value                                                                                                                            
        logs at or above this threshold go to stderr                                                                                                                       
  -v value                                                                                                                                             
        number for the log level verbosity
  -version
        Show version.
  -vmodule value
        comma-separated list of pattern=N settings for file-filtered logging
```

Reply:
Yes @Wangzihao18 you are right, that's why we have added the `CSIDriver` resource to support the newer versions of csi componenets.